redhat/rdma-core.spec

```diff
@@ -274,6 +274,13 @@ easy, object-oriented access to IB verbs.
 %setup
 
 %build
+# This package uses top level ASM constructs which are incompatible with LTO.
```

Comment:
In the commit message what is 'cherry picked from commit abad9da95aadc756bb7cc348da6e13509f499fdb'? Proper commit message plase

Reply:
Got it. I will update the commit message.